.github/workflows/builds.yml

```diff
@@ -81,26 +90,33 @@ jobs:
 
   linux_ubuntu:
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/ubuntu2004:v21
+    container: ghcr.io/acts-project/ubuntu2004:v22
     env:
       INSTALL_DIR: ${{ github.workspace }}/install
       ACTS_LOG_FAILURE_THRESHOLD: WARNING
     steps:
       - name: Install git lfs
-        run: apt install -y git-lfs
+        run: apt-get install -y git-lfs
 
       - uses: actions/checkout@v3
         with:
           submodules: true
           lfs: true
 
+      - name: Cache build
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace }}/ccache
+          key: ${{ runner.os  }}-ccache-linux_ubuntu_r1
```

Comment:
after seeing `r1` again - maybe it would be cool to have a global ENV variable that captures the ccache revision? since they will most likely roll the same time